code/client/clrcore/External/WeaponHash.cs

```diff
@@ -104,4 +104,34 @@ public enum VehicleWeaponHash
 		SearchLight = -844344963,
 		Radar = -764006018
 	}
-}
\ No newline at end of file
+
+	public enum AmmoType : ulong
+	{
+		Melee = 0,
+		FireExtinguisher = 1359393852,
+		Flare = 1808594799,
+		FlareGun = 1173416293,
+		PetrolCan = 18446744072810076321,
```

Comment:
18446744072810076321 -> `0xFFFFFFFFCA6318A1`
```suggestion
		PetrolCan = 0xCA6318A1,
```